curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/6633bdc3ca38c068c3735984.md

```diff
@@ -0,0 +1,217 @@
+---
+id: 6633bdc3ca38c068c3735984
+title: Step 19
+challengeType: 20
+dashedName: step-19
+---
+
+# --description--
+
+Add the x and y axes to the graph to obtain the following output:
+
+```py
+[
+    "⊣     ∙       ",
+    "⊣  ∙∙∙ ∙∙∙    ",
+    "⊣ ∙       ∙   ",
+    "⊣∙         ∙  ",
+    "⊣           ∙ ",
+    "⊣            ∙",
+    "⊣             ",
+    " ⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤",
+]
+```
```

Comment:
Is it just me, or this is not fully looking like it should?

Are the special characters messing up mono-spacing?

Reply:
Yes, they are messing up the mono-spacing, let me see if I find a different character...